ferrocene/ci/split-tasks.py

```diff
@@ -87,7 +87,7 @@ def __lt__(self, other):
 
         "oxidos": ["ferrocene-oxidos"],
 
-        "tools": ["rust-analyzer", "clippy", "rustfmt", "flip-link"],
+        "tools": ["clippy", "rustfmt", "flip-link"],
```

Comment:
is it a good guess that this will build r-a in different job, one that builds rustc